utils/kytConfig.js

```diff
@@ -4,12 +4,12 @@
 const path = require('path');
 const shell = require('shelljs');
 const mergeAll = require('ramda').mergeAll;
+const logger = require('../cli/logger');
```

Comment:
We can't do this. It's a chicken and egg problem. The logger references kytConfig. So in this file we just use the console.